### PR TITLE
Clean message section

### PR DESCRIPTION
**Pull request by FelixBlaisThon**
### Proposed Changes

Removed the message section in style
Added confirmation, warning and information prompt in modalConnected component

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/react-vapor/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/demo/src/components/examples/modalWindowExamples.tsx [hunk 1]
```diff
@@ -58,10 +59,87 @@ const ModalExampleDisconnected: React.FunctionComponent<ModalConnectedExamplePro
...
+    const modalInfoId = 'Modal-Info-Connected';
     const modalId = 'Modal-Connected-#1';
     return (
         <Section level={2} title="Modals connected to the redux store">
+            <Section level={3} title="Confirmation, warning and info prompts">
+                <Button
+                    className="btn"
+                    onClick={() => {
+                        return open(modalConfirmationId);
+                    }}
```
Comment (GermainBergeron) [inline packages/demo/src/components/examples/modalWindowExamples.tsx]:
```typescript
onClick={() => open(modalConfirmationId)}
```

File: packages/demo/src/components/examples/modalWindowExamples.tsx [hunk 2]
```diff
@@ -58,10 +59,87 @@ const ModalExampleDisconnected: React.FunctionComponent<ModalConnectedExamplePro
...
+                            </Button>
+                        </>
+                    }
+                    modalBodyClasses={['mod-header-padding', 'mod-form-top-bottom-padding']}
+                />
+                <Button
+                    className="btn"
+                    onClick={() => {
+                        return open(modalWarningId);
+                    }}
```
Comment (GermainBergeron) [inline packages/demo/src/components/examples/modalWindowExamples.tsx]:
```typescript
onClick={() => open(modalWarningId)}
```

File: packages/demo/src/components/examples/modalWindowExamples.tsx [hunk 3]
```diff
@@ -58,10 +59,87 @@ const ModalExampleDisconnected: React.FunctionComponent<ModalConnectedExamplePro
...
+                            Close
+                        </Button>
+                    }
+                    modalBodyClasses={['mod-header-padding', 'mod-form-top-bottom-padding']}
+                />
+                <Button
+                    className="btn"
+                    onClick={() => {
+                        return open(modalInfoId);
+                    }}
```
Comment (GermainBergeron) [inline packages/demo/src/components/examples/modalWindowExamples.tsx]:
```typescript
onClick={() => open(modalInfoId)}
```

File: packages/demo/src/components/examples/modalWindowExamples.tsx [hunk 4]
```diff
@@ -58,17 +59,74 @@ const ModalExampleDisconnected: React.FunctionComponent<ModalConnectedExamplePro
...
+            <Section level={3} title="Confirmation, warning and info prompts">
+                <Button className="btn" onClick={() => open(modalConfirmationId)}>
+                    Prompt confirmation
+                </Button>
+
+                <ModalCompositeConnected
+                    id={modalConfirmationId}
+                    title="Prompt confirmation"
+                    isPrompt
+                    classes={'mod-prompt'}
```
Comment (gdostie) [inline packages/demo/src/components/examples/modalWindowExamples.tsx]: I'm surprised that we have a `isPrompt` prop and that we still need to specify the `mod-prompt` class :thinking:

File: packages/demo/src/components/examples/modalWindowExamples.tsx [hunk 5]
```diff
@@ -58,17 +59,74 @@ const ModalExampleDisconnected: React.FunctionComponent<ModalConnectedExamplePro
...
+                    modalHeaderClasses={['mod-confirmation']}
+                    modalBodyChildren={<div className="mt2">{loremIpsum({count: 3})}</div>}
+                    modalFooterChildren={
+                        <>
+                            <Button className="btn mod-primary" onClick={() => handleClose(modalConfirmationId)}>
```
Comment (gdostie) [inline packages/demo/src/components/examples/modalWindowExamples.tsx]:
```suggestion
                            <Button primary small onClick={() => handleClose(modalConfirmationId)}>
```

File: packages/demo/src/components/examples/modalWindowExamples.tsx [hunk 6]
```diff
@@ -58,17 +59,74 @@ const ModalExampleDisconnected: React.FunctionComponent<ModalConnectedExamplePro
...
+                                Confirm
+                            </Button>
+                            <Button className="btn" onClick={() => handleClose(modalConfirmationId)}>
```
Comment (gdostie) [inline packages/demo/src/components/examples/modalWindowExamples.tsx]:
```suggestion
                            <Button small onClick={() => handleClose(modalConfirmationId)}>
```

File: packages/demo/src/components/examples/modalWindowExamples.tsx [hunk 7]
```diff
@@ -58,17 +59,74 @@ const ModalExampleDisconnected: React.FunctionComponent<ModalConnectedExamplePro
...
+
+                <ModalCompositeConnected
+                    id={modalWarningId}
+                    title="Prompt warning"
+                    isPrompt
+                    classes={'mod-prompt'}
+                    modalHeaderClasses={['mod-warning']}
+                    modalBodyChildren={<div className="mt2">{loremIpsum({count: 3})}</div>}
+                    modalFooterChildren={
+                        <Button className="btn" onClick={() => handleClose(modalWarningId)}>
```
Comment (gdostie) [inline packages/demo/src/components/examples/modalWindowExamples.tsx]:
```suggestion
                        <Button small onClick={() => handleClose(modalWarningId)}>
```

File: packages/demo/src/components/examples/modalWindowExamples.tsx [hunk 8]
```diff
@@ -58,17 +59,74 @@ const ModalExampleDisconnected: React.FunctionComponent<ModalConnectedExamplePro
...
+                            Close
+                        </Button>
+                    }
+                    modalBodyClasses={['mod-header-padding', 'mod-form-top-bottom-padding']}
+                />
+                <Button className="btn" onClick={() => open(modalInfoId)}>
```
Comment (gdostie) [inline packages/demo/src/components/examples/modalWindowExamples.tsx]:
```suggestion
                <Button small onClick={() => open(modalInfoId)}>
```

File: packages/demo/src/components/examples/modalWindowExamples.tsx [hunk 9]
```diff
@@ -58,17 +59,74 @@ const ModalExampleDisconnected: React.FunctionComponent<ModalConnectedExamplePro
...
+                </Button>
+                <ModalCompositeConnected
+                    id={modalInfoId}
+                    title="Prompt info"
+                    isPrompt
+                    classes={'mod-prompt'}
+                    modalHeaderClasses={['mod-info']}
+                    modalBodyChildren={<div className="mt2">{loremIpsum({count: 3})}</div>}
+                    modalFooterChildren={
+                        <Button className="btn" onClick={() => handleClose(modalInfoId)}>
```
Comment (gdostie) [inline packages/demo/src/components/examples/modalWindowExamples.tsx]:
```suggestion
                        <Button small onClick={() => handleClose(modalInfoId)}>
```

File: packages/demo/src/components/examples/modalWindowExamples.tsx [hunk 10]
```diff
@@ -58,17 +59,74 @@ const ModalExampleDisconnected: React.FunctionComponent<ModalConnectedExamplePro
...
+                />
+            </Section>
             <Section level={3} title="A simple modal window connected">
-                <Button
-                    className="btn"
-                    onClick={() => {
-                        return open(modalId);
-                    }}
-                >
+                <Button className="btn" onClick={() => open(modalId)}>
```
Comment (gdostie) [inline packages/demo/src/components/examples/modalWindowExamples.tsx]:
```suggestion
                <Button onClick={() => open(modalId)}>
```

File: packages/demo/src/components/examples/modalWindowExamples.tsx (hunk 4 above)
Comment (FelixBlaisThon) [reply packages/demo/src/components/examples/modalWindowExamples.tsx]: i tried it and it didn't change the modal.. :S

File: packages/demo/src/components/examples/modalWindowExamples.tsx (hunk 8 above)
Comment (FelixBlaisThon) [reply packages/demo/src/components/examples/modalWindowExamples.tsx]: pourquoi un small sur celui-là?

**Pull request merged**